### DISCUSSION TITLE
task user assignment

### DISCUSSION THREAD

**Discussion by joff13** [viewflow/viewflow#410] [General]
Dears,
Any suggestion on how to make possible to assign a user to a task, not tuntime but from a form?

example: the manager start a flow and assign the the flow itself or a specific task to a user

thanks for any suggestion
C

**Comment by kmmbvnr** [comment]
The BPMN approach assumes that you would record any human derision in a process data.

You need to store assigned user in a process data, and then return it from the  the .Assign(...) callable
